src/components/collapsibleContainer/CollapsibleContainer.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import * as $ from 'jquery';
+import { LinkSvg } from '../svg/LinkSvg';
+import { Svg } from '../svg/Svg';
+import { ITooltipProps, Tooltip } from '../tooltip/Tooltip';
+import { IClassName } from '../../utils/ClassNameUtils';
+import { JSXRenderable } from '../../utils/JSXUtils';
+
+export interface ICollapsibleContainerOwnProps {
+  id: string;
+  title: JSXRenderable;
+  expandedOnMount?: boolean;
+  informationUrl?: string;
+  informationTooltip?: ITooltipProps;
+  className?: IClassName;
+  collapsibleHeaderClassName?: IClassName;
+  collapsibleBodyClassName?: IClassName;
+}
+
+export interface ICollapsibleContainerStateProps {
+  expanded?: boolean;
+}
+
+export interface ICollapsibleContainerDispatchProps {
+  onMount?: () => void;
+  onUnmount?: () => void;
+  onToggleExpandedState?: (currentExpandedState: boolean) => void;
+}
+
+export interface ICollapsibleContainerProps extends
+  ICollapsibleContainerOwnProps,
+  ICollapsibleContainerStateProps,
+  ICollapsibleContainerDispatchProps { }
+
+export class CollapsibleContainer extends React.Component<ICollapsibleContainerProps> {
+  private toggleCollapsibleBodyOnExpandedStateChange(nextProps: ICollapsibleContainerProps) {
+    const collapsibleBodySelector = `#${this.props.id} .collapsible-body`;
+    const animationOptions = { duration: 350 };
+
+    if (this.props.expanded && !nextProps.expanded) {
+      $(collapsibleBodySelector).slideUp(animationOptions);
+    } else if (!this.props.expanded && nextProps.expanded) {
+      $(collapsibleBodySelector).slideDown(animationOptions);
```

Comment:
Let's try to use jquery only when absolutely necessary. You can easily use css transitions here. You can also use React's transitions https://reactjs.org/docs/animation.html.

Reply:
I did try with css and nothing worked as what we have in backbone/marionette...i guess you arrived at the same conclusion for the table collapsible rows where you used jquery as well? I would like you to consider leaving it as is here...this is the exact same behavior we have everywhere in the anim

Reply:
We could log another story for that too, i don't see it as a priority as of now. I can do that if you want.